src/live_restore/live_restore_fs.c

```diff
@@ -1657,6 +1657,48 @@ __validate_live_restore_path(WT_FILE_SYSTEM *fs, WT_SESSION_IMPL *session, const
     return (fh->close(fh, (WT_SESSION *)session));
 }
 
+/*
+ * __live_restore_copy_log_file --
```

Comment:
A general note, I have plans for this to become an atomic operation in line with how WiredTiger does atomic file write and copy but I don't think we need to implement that here.